demos/renderer/js/rendererDemo.js

```diff
@@ -204,7 +203,7 @@ var demo = demo || {};
      * and update the demo's model display.
      */
     var setupDataModel = function () {
-        var applier = fluid.makeChangeApplier(demo.data);
+        var applier = fluid.makeHolderChangeApplier({model: demo.data});
```

Comment:
Arguably orthogonal to this pull request, but this Renderer demo is immensely complex and full of code. Presumably a new demo is order some day.